packages/ETLoad.package/ETLPlaceholderObject.class/class/newFrom.and..st

```diff
@@ -0,0 +1,8 @@
+as yet unclassified
+newFrom: header and: bodyStream
```

Comment:
I believe a better style would be `newFromHeader: aHeader andBody: aBodyStream` but it's also probably fine as is.